pkg/sql/sqlstats/ssmemstorage/ss_mem_iterator.go

```diff
@@ -105,6 +106,20 @@ func (s *StmtStatsIterator) Next() bool {
 	return true
 }
 
+// copyData Copies the statement stat's data object under the mutex.
+// This function requires that there is a lock on the stats object.
+func copyData(stats *stmtStats) appstatspb.StatementStatistics {
+	stats.mu.AssertHeld()
```

Comment:
typically, a function that should be called with a lock taken also reflects that in the name, like `copyDataLocked`.